src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxes.class.php

```diff
@@ -95,10 +96,12 @@ public function GetHTML()
                       </div>';
 
             if ($hasEditPermissionForForeignTable) {
+
                 $url = $urlUtil->getArrayAsUrl(
                     array(
                         'tableid' => $oTargetTableConf->sqlData['id'],
-                        'pagedef' => 'tableeditor', 'id' => $recordId,
+                        'pagedef' => $this->getInputFilterUtil()->getFilteredGetInput('pagedef'),
```

Comment:
Could we
- move retrieval of the input filter util before the loop
- add the previous value `tableeditor` as default for `getFilteredGetInput` (or is this wrong in any case?)

Reply:
done

Reply:
We should then add the default for all such occurrences in this PR.

Reply:
right. Done.